src/Pages/Events/EventInfoModal.js

```diff
@@ -23,14 +22,12 @@ function EventInfoModal(props) {
         <Container className='center'>
           <Col>
             <Row className='event-date'>
-              <Ionicon icon='md-time' fontSize='1.5rem' color='#0098ab' />
               {getDateWithSlashes(currentEvent.eventDate.slice(0, 10))}{' '}
```

Comment:
import the function you created to show the clock, and render it here like below
```suggestion
          <Row className='event-location'>
            {clockSymbol()}
            {eventLocation}
          </Row>
```

Reply:
It says event-location is not defined

Reply:
what im saying is incorporate the functions you wrote